Core/src/org/sleuthkit/autopsy/rejview/RegeditExeValueFormatter.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Autopsy
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Copyright 2013 Willi Ballenthin
+ * Contact: willi.ballenthin <at> gmail <dot> com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.rejview;
+
+import com.williballenthin.rejistry.HexDump;
+import com.williballenthin.rejistry.RegistryParseException;
+import com.williballenthin.rejistry.ValueData;
+import java.io.UnsupportedEncodingException;
+import java.nio.ByteBuffer;
+import java.util.Iterator;
+import org.openide.util.NbBundle.Messages;
+
+/**
+ * Formats a ValueData to a string similar to Regedit.exe on Windows. For an
+ * example, see the nice sample here:
+ * http://raja5.files.wordpress.com/2009/08/wincleanup_regedit.jpg
+ */
+final class RegeditExeValueFormatter {
+
+    @Messages({"RegeditExeValueFormatter.valueNotSet.text=(value not set)"})
+    static String format(ValueData val) throws UnsupportedEncodingException, RegistryParseException {
+        StringBuilder sb = new StringBuilder();
+
+        switch (val.getValueType()) {
+            case REG_SZ: // empty case - intentional fall-through
+            case REG_EXPAND_SZ: {
+
+                String valString = val.getAsString();
+                if (valString.length() == 0) {
+                    sb.append(Bundle.RegeditExeValueFormatter_valueNotSet_text());
+                } else {
+                    sb.append(valString);
+                }
+                if (sb.length() > 48) {
```

Comment:
I think we should also make constants out of these, I would have no idea what 45 or 48 is if I had to debug the code.

Reply:
Alright, the constants might be a bit vague in name like "MAX_VALUE_STRING_LEN" or something like that but I agree even vaguely named constants would be easier to understand than the raw numbers.